src/test/generate-matrix.py

```diff
@@ -169,7 +169,12 @@ def __str__(self):
 )
 
 # Ubuntu: coverage
-matrix.add_build(name="coverage", coverage=True, jobs=2, args="--enable-pmix-bootstrap")
+matrix.add_build(
+    name="coverage",
+    coverage=True,
+    jobs=2,
+    args="--with-flux-security --enable-pmix-bootstrap",
+)
```

Comment:
Oh duh, sorry I didn't realize I was doing that!

Reply:
No, it is a problem with the way the script works (took me a second to figure out)